### ISSUE TITLE
Mouse cursor cannot reach area outside scaled display

### ISSUE THREAD

**Issue by Redsandro** [linuxmint/cinnamon#5392]
When scaling a bigger viewport on your display than its default resolution for reasons, the mouse cursor cannot reach the new screen estate.

Observed on Ubuntu Cinnamon 18 x64 on laptop.

To try this at home, scale your display by making the viewport bigger, in my case `LVDS1`:

```
$ xrandr --output LVDS1 --scale 1.2x1.2
```

Accidentally discovered when trying to recreate #5385 on simpler hardware.

**Comment by Redsandro** [comment]
To work around, use `xorg.conf` `metamodes` for scaling. E.g.:

```
DVI-I-0: 1280x1024 +0+0 {viewportin=1965x1572}
```

**Comment by Vahan86** [comment]
@Redsandro, is this still an issue for you with Cinnamon 4.2?

**Comment by Redsandro** [comment]
@Vahan86 I am not sure. I believe it was there 9 months ago, but I have since then retired the monitor and the video hardware. I cannot test #5385 related issues anymore.

Note that I didn't simply move on; the reason for replacing the setup was annoyance with a couple of long standing bugs with Cinnamon (or the upstream components responsible) in this configuration, so I'm not assuming anything. I recycled the small monitor. I only have the one 4K monitor left. And it doesn't scale beyond that.
